iOS/testbed/iOSTestbed/main.m

```diff
@@ -0,0 +1,20 @@
+//
+//  main.m
+//  iOSTestbed
+//
+
+#import <UIKit/UIKit.h>
+#import "AppDelegate.h"
+
+int main(int argc, char * argv[]) {
+    NSString * appDelegateClassName;
+    @autoreleasepool {
+        appDelegateClassName = NSStringFromClass([AppDelegate class]);
+
+        // iOS doesn't like uncaught signals.
+        signal(SIGPIPE, SIG_IGN);
+        signal(SIGXFSZ, SIG_IGN);
```

Comment:
I encountered this on Android as well. It turns out Python normally ignores these signals itself, but this is disabled by default with an isolated config. This can be fixed by adding the following line to the initialization code:

    config.install_signal_handlers = 1;

Then it should be possible to remove these manual ignores.